cocos/ui/UIEditBox/UIEditBox.cpp

```diff
@@ -123,6 +129,10 @@ bool EditBox::initWithSizeAndBackgroundSprite(const cocos2d::Size &size, cocos2d
         
         this->addTouchEventListener(CC_CALLBACK_2(EditBox::touchDownAction, this));
         
+        this->setPadding(_leftPadding, _topPadding, _rightPadding, _bottomPadding);
+        
+        this->setTextAlignment(_textHorizontalAlignMent, _textVerticalAlignMent);
```

Comment:
`_textHorizontalAlignMent` should be `_textHorizontalAlignment` and `_textVerticalAlignMent` should be `_textVerticalAlignment`